src/monaco.contribution.ts

```diff
@@ -9,6 +9,7 @@ import * as mode from './tsMode';
 import Emitter = monaco.Emitter;
 import IEvent = monaco.IEvent;
 import IDisposable = monaco.IDisposable;
+import { getVersion } from './languageFeatures';

```

Comment:
This ends up dragging the entire TS compiler to get loaded into the main thread.